Projects/11_EDA_Reduce_Hospital_Readmissions/11_Readmission.py

```diff
@@ -0,0 +1,96 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+import statsmodels.api as sm
+import plotly.plotly as py
+
+def load_and_clean():
+    column_names = ['Number of Discharges', 'Excess Readmission Ratio', 'Number of Readmissions', 'State']
+    df = hospital_read_df = pd.read_csv('data/cms_hospital_readmissions.csv', skip_blank_lines = True)
+    df = df[column_names]
+    df.columns=['Discharges', 'Excess_Ratio', 'Readmissions', 'State']
+    df = df.dropna()
+    df.Discharges = df.Discharges.astype(float)
+    df = df.iloc[np.isreal(df.Discharges),:]
+    return df
+    
+def myscatter(df):
+    plt.figure(figsize=(12,7))
+    plt.subplot(1,3,1);
+    plt.scatter(df.Discharges.apply(np.log10), df.Readmissions.apply(np.log10), c=df.Excess_Readmission_Ratio)
+    plt.colorbar()
+    plt.xlabel('Log 10 of Discharges')
+    plt.ylabel('Log 10 of Readmissions')
+    plt.title('Excess Readmission Ratio')
+
+    plt.subplot(1,3,2);
+    bool_ = df.Excess_Readmission_Ratio < 1.0
+    plt.scatter(df.Discharges.apply(np.log10)[bool_], df.Readmissions.apply(np.log10)[bool_], c=df.Excess_Readmission_Ratio[bool_])
```

Comment:
You are calculating log for a number of rows that you'll later filter out. Can you do the filtering earlier?